sdk/core/Azure.Core/perf/EventSourceBenchmark.cs

```diff
@@ -56,16 +56,17 @@ public void CleanUp()
             _eventSource.Dispose();
         }
 
-        [Benchmark(Baseline = true, Description = "Old implementation")]
+        // Benchmarks
+        [Benchmark(Description = "Old implementation, includes reformatting")]
         public void OldImplementation_CallFormatting()
         {
-            _eventSource.RequestOld(Sanitizer.SanitizeUrl(_uri.ToString()), _iteration++, _iteration, FormatHeaders());
+            _eventSource.RequestOld(Sanitizer.SanitizeUrl(_uri.ToString()), _iteration++, _iteration, _headersBytes);
```

Comment:
does the header formatting need to be kept too in addition to the sanitization call?

Reply:
If there wasn't a specific reason that you wanted to test the scenario without it, I think it would make sense to leave it in,